geom.go

```diff
@@ -80,6 +80,36 @@ func (g *GeoM) Element(i, j int) float64 {
 	}
 }
 
+// A returns the A value of the matrix.
```

Comment:
`// A returns the element at the first row and the first column of the matrix` and so on.